kernel/fablib/ssh.go

```diff
@@ -67,7 +67,9 @@ func RemoteShell(factory SshConfigFactory) error {
 	if err != nil {
 		return err
 	}
-	fd := int(os.Stdin.Fd())
+
+	fd := int(os.Stdout.Fd())
```

Comment:
`stdin` can't be used as a handle for subsequent calls in windows. Should be semantically equivalent.

Reply:
Need to make sure we test this on Linux.